### ISSUE TITLE
Forbidden (403) CSRF verification failed. Request aborted.

### ISSUE THREAD

**Issue by surajram09** [DefectDojo/django-DefectDojo#7439]
### Discussed in https://github.com/DefectDojo/django-DefectDojo/discussions/7409

<div type='discussions-op-text'>

<sup>Originally posted by **vinod-boda** January 12, 2023</sup>
Hi Team,

We are implementing Defectdojo within our organization. Our team is facing "Forbidden (403) CSRF verification failed. Request aborted" while implementing SAML login. Can someone help on fixing this issue?

<img width="624" alt="Forbidden403" src="https://user-images.githubusercontent.com/115187343/212097856-a58e5dff-e46a-4a77-8a92-79bd4da06ccd.PNG">
</div>

**Comment by dsever** [comment]
Are you doing kind of rewrite, like you have WAF in front that servers backends with different fqdn?

**Comment by surajram09** [comment]
No, we have DefectDojo on a Openshift environment on a Openshift route.

**Comment by surajram09** [comment]
[Configmap.txt](https://github.com/DefectDojo/django-DefectDojo/files/10450180/Configmap.txt)

This is what our settings.dist.py looks like.

**Comment by QueTheWizard** [comment]
I'm experiencing the same issue as well, trying to implement OneLogin SSO.

**Comment by oluapG** [comment]
Can you try include the url  "sso.issban.dev...." in variable DD_CSRF_TRUSTED_ORIGINS ?

**Comment by devGregA** [comment]
Closing — this SAML/CSRF 403 is resolved by adding your SSO origin to `DD_CSRF_TRUSTED_ORIGINS` (see above). Please reopen if it persists on a current version.
